README.md

```diff
@@ -54,6 +54,8 @@ For these tasks you may prefer using webpki in combination with libraries like
 Changelog
 =========
 
+* 0.101.1 (2023-07-05)
+  - Fixed 32bit architecture compatibility.
```

Comment:
Style nit: I would usually spell this "32-bit".

Reply:
:+1: Fixed!